Add an empty FAQ, and update menus

kinow opened the pull request:
Hi again @wcolen 

Here's the updated menu, an empty FAQ page (to be updated later as part of OPENNLP-504).

Top menu items have icons. There is code to highlight items active. So the item in the drop down menu will be blue, and the top level menu item active will be gray.

Attached simple screenshot of some menu items.

![1-fullpage](https://cloud.githubusercontent.com/assets/304786/25219355/5539deae-2602-11e7-8fbb-28eda0af4c4d.png)

![2-fullpage](https://cloud.githubusercontent.com/assets/304786/25219357/596f7830-2602-11e7-90ae-cc56f6e0eaa2.png)

![3-fullpage](https://cloud.githubusercontent.com/assets/304786/25219367/5c8ca1c8-2602-11e7-92a3-37357eb9720d.png)

![4-fullpage](https://cloud.githubusercontent.com/assets/304786/25219368/5f475cfa-2602-11e7-9448-03c5755d3f4b.png)

![5-fullpage](https://cloud.githubusercontent.com/assets/304786/25219373/61ccb1be-2602-11e7-8f17-e814ac4a3437.png)

And a screenshot with the mobile responsive version.

Cheers
Bruno

wcolen commented:
Can you improve the footer? Maybe centerer? The two buttons in the middle (get starterd, download), would look better if not together, same size, maybe a custom color from the logo.

kinow commented:
>Can you improve the footer? Maybe centerer?

Sure thing. Does it look better now? Spent quite some time trying to centre the text and also fix the hidden content due to resize... then realized that Jena's website worked fine. Simply fixed our CSS using their style (thanks to whoever found a good combination of CSS styles for Jena's footer!) and I think it looks better now.

![opennlp-20170421-fullpage](https://cloud.githubusercontent.com/assets/304786/25273009/1b4966ce-26de-11e7-8890-de980d0cc871.png)

![opennlp-mobile-20170421-fullpage](https://cloud.githubusercontent.com/assets/304786/25273015/1f016320-26de-11e7-8b76-7703908bd35a.png)

>The two buttons in the middle (get starterd, download), would look better if not together, same size, maybe a custom color from the logo.

Easy-peasy. Done in the screen shot above too.

Plus:

* Added favicon
* Added a privacy-policy page, for the footer link (was pointing to opennlp.apache.org/privacy-policy.html, now to /privacy-policy.html)
* Updated icon to include the TM for trademark (Thanks Joern!)

Cheers
Bruno

kinow commented:
Committed to my branch, so pull request automatically updated too :+1:
